openmp/tools/omptest/CMakeLists.txt

```diff
@@ -0,0 +1,137 @@
+##===----------------------------------------------------------------------===##
+#
+# Build OMPT unit testing library: ompTest
+#
+##===----------------------------------------------------------------------===##
+
+cmake_minimum_required(VERSION 3.22)
+project(omptest LANGUAGES CXX)
+
+option(LIBOMPTEST_BUILD_STANDALONE
+       "Build ompTest 'standalone', i.e. w/o GoogleTest."
+       ${OPENMP_STANDALONE_BUILD})
+option(LIBOMPTEST_BUILD_UNITTESTS
+       "Build ompTest's unit tests, requires GoogleTest." OFF)
```

Comment:
OpenMP options must start with `OPENMP_`. llvm/runtimes/CMakeLists.txt automatically forwards options starting with the runtimes' names to the `runtimes/runtimes-bins` nested build. It doesn't to that for `LIBOMPTEST_`